package.json

```diff
@@ -9,9 +9,10 @@
     "compile": "webpack",
     "karma": "karma start karma.config.js",
     "lint": "eslint '**/*.js'",
+    "lint-fix": "eslint '**/*.js' --fix",
     "prepublish": "yarn compile",
     "start": "webpack-dev-server",
-    "test": "yarn ci:lint && yarn ci:test",
+    "test": "yarn ci:test && yarn ci:lint",
```

Comment:
As a part of adopting a linter, we need to get into using them since it's a part of our dev flow. Otherwise, I believe we'll lose sight of what we consider quality code since whatever rules we're using to lint will take the backseat when it should be a first class citizen when it comes to producing code.